src/server_manager/web_app/app.ts

```diff
@@ -423,10 +423,6 @@ export class App {
         }
       }
       await this.updateServerView(server);
```

Comment:
I think you also need to call updateServerView inside the if-block above, before waitOnInstall, so that the loading view is selected.

Reply:
Done. Although I think it's not needed because we call `showServer` immediately after `addServer` and that takes care of showing the correct screen by calling `updateServerView`. 

It might be a good idea to remove the call to `updateServerView` in `addServer` entirely. We really only depend on the call to `isHealthy`. The only two places we call updateServerView are in `addServer` and `showServer`, so that change would simplify the logic even further imo.

Reply:
Refactored